src/doc/html_templates/html_templates.nit

```diff
@@ -319,6 +319,15 @@ redef class DocComposite
 		end
 		lst.add_li new ListItem(content)
 	end
+
+	# ID used in HTML tab labels.
+	#
+	# We sanitize it for Boostrap JS panels that do not like ":" and "." in ids.
+	var html_tab_id: String is lazy do
+		var id = html_id.replace(":", "")
+		id = id.replace(".", "")
```

Comment:
Do you think it is worth that you replace these chars with html entities like `&something;` to avoid conflict on ids?